packages/plugins/src/dashboard-components/layers/absolute-position-layout.js

```diff
@@ -21,16 +26,35 @@ export default class AbsolutePositioningLayout extends Layer {
     dashboard.subscribe('elementSelect', () => {
       if (this.#selectedElement) {
         this.#addResizeInteraction();
+        this.#addDragInteraction();
+      }
+    });
+
+    window.addEventListener('keydown', (e) => {
+      if(e.key == 'Shift') {
+        this.#selectionBox.style.border = '2px solid blue'
+        this.snappingEnabled = true;
       }
     });
 
+    window.addEventListener('keyup', (e) => {
+      if(e.key == 'Shift') {
+
+        this.#selectionBox.style.border = '2px dashed green'
```

Comment:
I'm surprised a `;` wasn't automatically added to this since I set up eslint to fix up code on precommit. I probably haven't set it up properly. Also, I need to add prettier.